analyzer/runtime/extract.go

```diff
@@ -20,9 +20,10 @@ import (
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/accounts"
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/consensusaccounts"
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/core"
-	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/evm"
+	evm_sdk "github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/evm"
```

Comment:
I've been using "sdk" as a prefix, e.g. in sdkTypes. would sdkEVM be appropriate here?

Reply:
yeah that's more in line with the other imports, thanks, renamed